tests/unit/test_mysqlsh_helpers.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import subprocess
+import unittest
+from unittest.mock import PropertyMock, patch
+
+from mysqlsh_helpers import MySQL, MySQLInstanceConfigureError
+
+
+class TestMySQL(unittest.TestCase):
+    def setUp(self):
+        root_password = "password"
+        cluster_admin_user = "clusteradmin"
+        cluster_admin_password = "innodb"
+        instance_address = "127.0.0.1"
+
+        self.mysql = MySQL(
+            root_password, cluster_admin_user, cluster_admin_password, instance_address
+        )
+
+    @patch("os.path.exists", return_value=False)
+    @patch("mysqlsh_helpers.MySQL.mysqlsh_bin", new_callable=PropertyMock)
+    @patch("subprocess.check_call")
+    @patch("tempfile.NamedTemporaryFile")
+    @patch("subprocess.check_output")
+    def test_run_mysqlsh_script(
```

Comment:
Hm, is anything really being tested here? All the calls are mocked so nothing real is happening. No suggestions though, I understand that this is needed to get coverage up, but sometimes it feels unnecessary.

Reply:
good call. removed the superfluous tests in 46822799e0a54478e31361bb4cbea6fd70c4e454